File: app/src/main/java/com/kieran/winnipegbusbackend/TripPlanner/classes/WalkSegment.java
package com.kieran.winnipegbusbackend.TripPlanner.classes;

import org.json.JSONObject;

public class WalkSegment extends Segment {
    public WalkSegment(JSONObject segment, String fromTitle, String toTitle) {
        super(segment, fromTitle, toTitle);
    }

    @Override
    public String toString() {
        if(from != null && to != null)
            return super.toString();
        else
            return "Walk";
    }
}

File: app/src/main/java/com/kieran/winnipegbusbackend/TripPlanner/classes/WalkSegment.kt
package com.kieran.winnipegbusbackend.TripPlanner.classes

import org.json.JSONObject

class WalkSegment(segment: JSONObject) : Segment(segment) {

    override fun toString(): String {
        return if (from != null && to != null)
            super.toString()
        else
            "Walk"
    }
}
